### PR TITLE
Update expressionParser.js: do not log as warn contextAvailable

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- FIX: do not log as warn invalid context in contextAvailable of expression plugion legacy
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- FIX: change severity from WARN to INFO when invalid context in contextAvailable of legacy expression plugin occcurs
```

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:1]: No associate issue?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: No issue number

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**